docs/howto/retry.rst

```diff
@@ -34,11 +34,20 @@ Simple strategies
 Advanced strategies
 ^^^^^^^^^^^^^^^^^^^
 
-You can get a more precise strategy using a :py:class:`RetryStrategy` instance::
+Advanced startegies let you:
+- define a maximum number of retries
+- define the retry delay, with constant, linear and exponential backoff options
+- define the exception types you want to retry on
```

Comment:
We should point out that defining the exception types is optional, and that if not given, we will retry on every type of exception I think :O